core/src/components/progress-bar/test/buffer/progress-bar.e2e.ts

```diff
@@ -0,0 +1,25 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs().forEach(({ title, screenshot, config }) => {
+  test.describe(title('progress-bar: buffer'), () => {
+    test.describe('with a dynamic progress value', () => {
```

Comment:
Thoughts on merging the new `buffer` and `determinate` tests into one `dynamic-value` test? All the other progress bars are duplicates from the `basic` test.